content/sponsors.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sponsors For TARDIS
+seo:
+  title: Sponsors for TARDIS
+  description: Sponsors for TARDIS Organisation
```

Comment:
Add the 1st sentence of Sponsors page here

And everywhere below for description